chidioguejiofor/InitialisePythonApps
# #164838842: Creates pre-commit hook

#### What does this PR achieve?
- Initialises the repo and set up pre-commit hooks

#### Any background context?
- Setups pipenv shell 
- installs `yapf` package

#### Manual Testing Steps
- Add a python file 
- Add code that messes up the python file 
- Try to commit those changes
- Commit hook should fix the linting issues

#### Screenshots:
- None
